test/parallel/test-dgram-send-empty-array.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const dgram = require('dgram');
+
+const client = dgram.createSocket('udp4');
+
+const timer = setTimeout(function() {
+  throw new Error('Timeout');
+}, common.platformTimeout(200));
```

Comment:
Yes, but these kind of timers have been source of flaky tests. I don't think relying in the test runner timeout is a bad idea.